src/debugger/mock-ui/debugger-cli.js

```diff
@@ -51,38 +46,47 @@ function readCLIArguments(process, console): DebuggerCLIArguments {
     arg = arg.slice(2);
     if (arg === "adapterPath") {
       adapterPath = args.shift();
-    } else if (arg === "prepack") {
-      prepackCommand = args.shift();
-    } else if (arg === "inFilePath") {
-      inFilePath = args.shift();
-    } else if (arg === "outFilePath") {
-      outFilePath = args.shift();
+    } else if (arg === "prepackRuntime") {
+      prepackRuntime = args.shift();
+    } else if (arg === "prepackArguments") {
+      prepackArguments = args.shift().split(" ");
+    } else if (arg === "sourceFile") {
+      sourceFile = args.shift();
+    } else if (arg === "debugInFilePath") {
+      debugInFilePath = args.shift();
+    } else if (arg === "debugOutFilePath") {
+      debugOutFilePath = args.shift();
     } else {
       console.error("Unknown argument: " + arg);
       process.exit(1);
     }
   }
-  if (inFilePath === 0) {
-    console.error("No input file path provided!");
+  if (debugInFilePath === 0) {
```

Comment:
Why are we comparing string against numeric 0 here?

Reply:
should be a length check. will fix.